src/services/auth.ts

```diff
@@ -3,40 +3,34 @@ import PublicClientApplication, { MSALAccount, MSALResult, MSALSilentParams } fr
 import type {
   MSALConfiguration,
 } from "react-native-msal";
-import { Platform } from "react-native";
-
-//import { Platform } from "react-native";
 
 export let pca: PublicClientApplication | null = null;
 
-export async function msalInit(clientId: string, redirectUri: string, authority?: string) {
-
+export async function msalInit(authority: string, clientId: string, redirectUri: string) {
   const config: MSALConfiguration = {
     auth: {
+        authority: authority,
```

Comment:
Do we need to make authority a required argument? Most, if not all apps, use the same authority

Reply:
Yeah. Switching it back to optional as It will be easier for Equinor-projects as you mentioned, but remain usable by the public if they pass in the `authority`-prop👌 If Equinor is changing authority we would just have to change the default authroty prop in `mad-expo-core` and bump version of all our apps and make a new build.